src/lib.rs

```diff
@@ -152,8 +152,8 @@ pub mod utils;
 #[cfg(feature = "validate")]
 #[cfg_attr(docsrs, doc(cfg(feature = "validate")))]
 pub mod validate;
-#[cfg(feature = "verify")]
-#[cfg_attr(docsrs, doc(cfg(feature = "verify")))]
+#[cfg(any(all(feature = "verify", not(feature = "verify-aws-lc")), all(not(feature = "verify"), feature = "verify-aws-lc")))]
```

Comment:
The pattern I was aiming for is so that it would default to one but switch to the other if it was explicitly stated, but I couldn't figure out how to make that work.